cli/ansible/wf-vpnc-setup.yml

```diff
@@ -1,3 +1,5 @@
+- import_playbook: wf-ensure-python.yml
```

Comment:
Remove this line
`wf-ensure-python.yml` should be on demand. And in future, any ansible environment setup will not be in any specific deployment app bundle

Reply:
This task is quite safe on any hosts and there should be no impact to target environment. I still think it's safe to keep it on top of the playbook.

Reply:
It is not related to safe or no safe.
I emphasis once again: VPN is one of many applications will be deployed on target environment. Keep it clean and single responsibility only and only for VPN. Don't touch any general configuration on environment that doesn't relate directly to VPN task.
Tweak Ansible/Python environment for remote deployment is unrelated to VPN task. This task will be in common tasks in Ansible base image.

Just simple keep workflow in this repository until I provide Ansible base image, and make a call manually in first time of your test environment.